language/scripts/setup_env.py

```diff
@@ -138,13 +138,32 @@ def build_terra(terra_dir, llvm_dir, cache, is_cray, thread_count):
         cwd=terra_dir,
         env=env)
 
+def build_hdf(source_dir, install_dir, thread_count, is_cray):
+    env = None
+    if is_cray:
+        env = dict(list(os.environ.items()) + [
+            ('CC', os.environ['HOST_CC']),
+            ('CXX', os.environ['HOST_CXX']),
+        ])
+    subprocess.check_call(
+        ['./configure',
+         '--prefix=%s' % install_dir,
+         '--enable-threadsafe',
+         '--disable-hl'],
+        cwd=source_dir,
+        env=env)
+    subprocess.check_call(['make', '-j', str(thread_count)], cwd=source_dir)
+    subprocess.check_call(['make', 'install'], cwd=source_dir)
+
 def build_regent(root_dir, use_cmake, cmake_exe,
-                 gasnet_dir, llvm_dir, terra_dir, conduit, thread_count):
+                 gasnet_dir, llvm_dir, terra_dir, hdf_dir, conduit, thread_count):
     env = dict(list(os.environ.items()) + [
         ('CONDUIT', conduit),
         ('GASNET', gasnet_dir),
         ('USE_GASNET', os.environ['USE_GASNET'] if 'USE_GASNET' in os.environ else '1'),
         ('LLVM_CONFIG', os.path.join(llvm_dir, 'bin', 'llvm-config')),
+        ('USE_HDF', os.environ['USE_HDF'] if 'USE_HDF' in os.environ else '1'),
```

Comment:
I'm not sure we want this enabled by default. I'm guessing we could probably let the user turn this on with a flag like `--hdf` or something. In which case this conditional would turn into `os.environ['USE_HDF'] if 'USE_HDF' in os.environ else ('1' if use_hdf else '0')`

Reply:
Actually I'd rather remove line 165 altogether, and avoid adding another option to setup_env.py. This will cause install.py to read the value of USE_HDF from the environment directly. In that scipt, HDF is off by default, unless USE_HDF=1.

Reply:
That's fine.

Reply:
Done